src/components/Group/Navbar.vue

```diff
@@ -9,6 +9,20 @@
       class="col"
       ref="groupnav"
     >
+      <div class="">
+        <q-transition
+          duration="510"
+          name="turn-in"
+          appear
+        >
+          <div class="photoCard shadow-2">
+            <ProfilePicture
+              :group="group"
```

Comment:
The changes in Navbar.vue seem unrelated to the referenced issue. What was your intention here?
Also,`ProfilePicture` doesn't have a `group` prop, but needs a `user` prop, so this code seems unlikely to work.